feature/resume/impl/src/main/java/com/example/impl/step3/ResumeStep3Screen.kt

```
@@ -0,0 +1,4 @@
+package com.example.feature.resume.impl.step3
+
+class ResumeStep3Screen {
+}
```

detekt (1.23.8)

[warning] The class or object ResumeStep3Screen is empty.

(detekt.empty-blocks.EmptyClassBlock)